.github/workflows/continuous-integration.yml

```diff
@@ -1,17 +1,20 @@
-name: Continuous Integration
+name: Run continuous integration
 
 on:
   push:
     branches-ignore:
-      - master
```

Comment:
Do we want CI to run on merges into master? It would have already run on the feature branch, but if we're concerned about potential integration failures then I suppose it makes sense.

Reply:
I think you're right and prefer to not have to do redundant work. There shouldn't be a point in running the CI twice.